static/modes/llvm-ir-mode.js

```diff
@@ -0,0 +1,140 @@
+            'fptosi', 'fptoui', 'fptrunc', 'free', 'frem', 'fsub', 'getelementptr',
+            'icmp', 'inbounds', 'indirectbr', 'insertelement', 'insertvalue',
+            'inttoptr', 'invoke', 'landingpad', 'load', 'lshr', 'malloc', 'max', 'min',
+            'mul', 'nand', 'ne', 'ninf', 'nnan', 'nsw', 'nsz', 'nuw', 'oeq', 'oge', 'ogt', 'ole',
+            'olt', 'one', 'or', 'ord', 'phi', 'ptrtoint', 'resume', 'ret', 'sdiv', 'select',
+            'sext', 'sge', 'sgt', 'shl', 'shufflevector', 'sitofp', 'sle', 'slt', 'srem',
+            'store', 'sub', 'switch', 'trunc', 'udiv', 'ueq', 'uge', 'ugt', 'uitofp', 'ule', 'ult',
+            'umax', 'umin', 'une', 'uno', 'unreachable', 'unwind', 'urem', 'va_arg',
+            'xchg', 'xor', 'zext'
+        ],
+        // llvmKeyword
+        keywords: [
+            'acq_rel', 'acquire', 'addrspace', 'alias', 'align', 'alignstack', 'alwaysinline',
+            'appending', 'argmemonly', 'arm_aapcscc', 'arm_aapcs_vfpcc', 'arm_apcscc', 'asm',
+            'atomic', 'available_externally', 'blockaddress', 'builtin', 'byval', 'c', 'catch',
+            'caller', 'cc', 'ccc', 'cleanup', 'coldcc', 'comdat', 'common', 'constant',
+            'datalayout', 'declare', 'default', 'define', 'deplibs', 'dereferenceable',
+            'distinct', 'dllexport', 'dllimport', 'dso_local', 'dso_preemptable', 'except',
+            'external', 'externally_initialized', 'extern_weak', 'fastcc', 'filter', 'from',
+            'gc', 'global', 'hhvmcc', 'hhvm_ccc', 'hidden', 'initialexec', 'inlinehint',
+            'inreg', 'inteldialect', 'intel_ocl_bicc', 'internal', 'linkonce', 'linkonce_odr',
+            'localdynamic', 'localexec', 'local_unnamed_addr', 'minsize', 'module',
+            'monotonic', 'msp430_intrcc', 'musttail', 'naked', 'nest', 'noalias', 'nobuiltin',
+            'nocapture', 'noimplicitfloat', 'noinline', 'nonlazybind', 'nonnull', 'norecurse',
+            'noredzone', 'noreturn', 'nounwind', 'optnone', 'optsize', 'personality',
+            'private', 'protected', 'ptx_device', 'ptx_kernel', 'readnone', 'readonly',
+            'release', 'returned', 'returns_twice', 'sanitize_address', 'sanitize_memory',
+            'sanitize_thread', 'section', 'seq_cst', 'sideeffect', 'signext', 'syncscope',
+            'source_filename', 'speculatable', 'spir_func', 'spir_kernel', 'sret', 'ssp',
+            'sspreq', 'sspstrong', 'strictfp', 'swiftcc', 'tail', 'target', 'thread_local',
+            'to', 'triple', 'unnamed_addr', 'unordered', 'uselistorder', 'uselistorder_bb',
+            'uwtable', 'volatile', 'weak', 'weak_odr', 'within', 'writeonly', 'x86_64_sysvcc',
+            'win64cc', 'x86_fastcallcc', 'x86_stdcallcc', 'x86_thiscallcc', 'zeroext'
+        ],
+
+        escapes: /\\(?:[abfnrtv\\"']|x[0-9A-Fa-f]{1,4}|u[0-9A-Fa-f]{4}|U[0-9A-Fa-f]{8})/,
+
+        tokenizer: {
+            root: [
+                [/[,(){}<>\[\]]/, 'delimiters'],
```

Comment:
The `\` before the `[` is unnecessary (and the IDE I use gives it a wiggly red underline. It seems OK to change this to:

```
                [/[,(){}<>[\]]/, 'delimiters'],
```

Reply:
Indeed, because we're already inside `[]`.

```suggestion
                [/[,(){}<>[\]]/, 'delimiters'],
```

Reply:
I'd probably keep them: it's expressive and for all I know it might help the editor out in some way (maybe not now, but in future?). We're looking to upgrade Monaco in a separate PR.